client/state/purchases/test/reducer.js

```diff
@@ -45,19 +44,19 @@ describe( 'reducer', () => {
 	it( 'should return an object with the list of purchases when fetching completed', () => {
 		let state = reducer( undefined, {
 			type: PURCHASES_USER_FETCH_COMPLETED,
-			purchases: [ { id: 1, siteId, userId }, { id: 2, siteId, userId } ]
+			purchases: [ { ID: 1, blog_id: siteId, user_id: userId }, { ID: 2, blog_id: siteId, user_id: userId } ]
```

Comment:
Unfortunately, the API does not return IDs of type `Number` ( 😞 ), so now that `purchases` here represents content from the API instead of content from the assembler, `getByPurchaseId` will only select a purchase if used with a string ID.